### PR TITLE
ADMIN PR - New ondatra and gosnappi version

### PR DESCRIPTION
**Pull request by octpetre**
- Latest ondatra which allows LAG group reservations
- Latest otg version (smaller footprint) and its compatible gosnappi
- Added juniper config and textproto
- Added arista_ixia_lag textproto
- Added lag testbed file

### CODE REVIEW

**Review by liulk — CHANGES_REQUESTED** [review]
Sorry I forgot to send out my pending review.

File: topologies/atedut_lag.testbed [hunk 1]
```diff
@@ -0,0 +1,116 @@
+duts {
```
[the remark this answers is not in the document]
Comment (octpetre) [reply topologies/atedut_lag.testbed:1]: This is the ondatra testbed file and is not related to the kne topology file (topologies/kne/arista_ixia_lag.textproto). All the testbed files are under topologies. Are you sure you want it moved under topologies/kne ?

[the remark this answers is not in the document]
Comment (liulk) [reply topologies/atedut_lag.testbed:1]: The reason I asked for the move is that this testbed uses the group field which is kne specific. Tests against real hardware would not be using this testbed.

[the remark this answers is not in the document]
Comment (octpetre) [reply topologies/atedut_lag.testbed:1]: Agreed! Moved under topologies/kne

File: topologies/atedut_lag.testbed [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+ates {
+  id: "ate"
+  vendor: IXIA
+  ports {
+    id: "port1"
+  }
+  ports {
+    id: "port2"
+    group: "lag"
```
[the remark this answers is not in the document]
Comment (anjan-keysight) [reply topologies/atedut_lag.testbed]: @liulk  Raised separate PR for the documentation.
https://github.com/openconfig/ondatra/pull/38
@octpetre FYI

**Review by liulk — APPROVED** [review]

File: topologies/kne/arista_ixia.textproto [hunk 3]
```diff
@@ -39,7 +39,7 @@ nodes: {
         config_path: "/mnt/flash"
         config_file: "startup-config"
         file: "arista_ceos.config"
-    	image: "us-central1-docker.pkg.dev/kt-nts-athena-dev/keysight/ceos:4.29.0F-28265717"
+    	image: "us-central1-docker.pkg.dev/kt-nts-athena-dev/keysight/ceos:4.29.0F-28493660"
```
Comment (marcushines) [inline topologies/kne/arista_ixia.textproto:42]: we should never check images from private repos

this should just be "ceos:latest"

and it is then up to the runner of this to either provide that image or substitute it

File: topologies/kne/arista_ixia_lag.textproto [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+            self_signed: {
+                cert_name: "gnmiCert.pem",
+                key_name: "gnmiCertKey.pem",
+                key_size: 4096,
+            }
+        }
+        config_path: "/mnt/flash"
+        config_file: "startup-config"
+        file: "arista_ceos.config"
+    	image: "us-central1-docker.pkg.dev/kt-nts-athena-dev/keysight/ceos:4.29.0F-28493660"
```
Comment (marcushines) [inline topologies/kne/arista_ixia_lag.textproto:90]: same

File: topologies/kne/juniper_ixia.textproto [hunk 5]
```diff
@@ -0,0 +1,98 @@
+name: "juniper-ixia"
+nodes: {
+    name: "cptx1"
+    type: JUNIPER_CEVO
+    vendor: JUNIPER
+    model: "cptx"
+    os: "evo"
+    config: {
+      image: "cptx:21.2X80-D25.4"
```
Comment (marcushines) [inline topologies/kne/juniper_ixia.textproto:9]: "cptx:latest"

**Review by liulk — CHANGES_REQUESTED** [review]
Please addresses Marcus' comment?

File: topologies/kne/juniper_ixia.textproto (hunk 5 above)
Comment (octpetre) [reply topologies/kne/juniper_ixia.textproto:9]: Done

File: topologies/kne/arista_ixia_lag.textproto (hunk 4 above)
Comment (octpetre) [reply topologies/kne/arista_ixia_lag.textproto:90]: Done

File: topologies/kne/arista_ixia.textproto (hunk 3 above)
Comment (octpetre) [reply topologies/kne/arista_ixia.textproto:42]: I will change to "latest"

**Review by liulk — APPROVED** [review]

**Comment by liulk** [comment]
@bstoll does the disco-idea failure have anything to do with the changes introduced in this PR?

**Review by liulk — APPROVED** [review]

**Pull request merged**